torchvision/datasets/ucf101.py

```diff
@@ -14,7 +14,10 @@ class UCF101(VisionDataset):
     UCF101 is an action recognition video dataset.
     This dataset consider every video as a collection of video clips of fixed size, specified
     by ``frames_per_clip``, where the step in frames between each clip is given by
-    ``step_between_clips``.
+    ``step_between_clips``. The dataset itself can be downloaded from `here
+    <https://www.crcv.ucf.edu/data/UCF101/UCF101.rar>`; annotations that ``annotation_path``
+    should be pointing to can be downloaded from `here
```

Comment:
I think in general it's best practice not to directly link to a downloadable file, but instead explain how to download it. The link might change and break, but the instructions should be a bit more robust. For example, WDYT of saying something like:

> The annotation files contains the train/test splits information and can correspond to the "Train/Test Splits for Action Recognition on UCF101 data set" link on the home page of the dataset.

Reply:
Also, in the description for `annotation_path` I think we should  say something like

> see docstring above for download instructions of these files